tests/test_files/sim_config.yaml

```diff
@@ -0,0 +1,37 @@
+  molecular_model:
```

Comment:
include units in variable names, and incorporate comment info if not too verbose.